graphprot/graphprot_seqmodel

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+DESCRIPTION = """
+GraphProt:
+
+Explicit Decomposition with Neighborhood (EDeN) utility program for
+sequence-based modeling of binding preferences of RNA-binding proteins.
+"""
+
+EPILOG = """
+Author: Daniel Maticzka
+Copyright: 2015
+License: GPL
+Maintainer: Daniel Maticzka
+Email: maticzkd@informatik.uni-freiburg.de
+Status: Development
+
+Please cite:
+
+    Daniel Maticzka, Sita J. Lange, Fabrizio Costa, Rolf Backofen
+    GraphProt: modeling binding preferences of RNA-binding proteins
+    Genome Biology, 2014, 15(1), R17
+"""
+
+import os
+import logging
+
+from numpy.random import randint, uniform
+from sklearn.linear_model import SGDClassifier
+from eden.graph import Vectorizer
+from eden.model_base import ModelInitializerBase, main_script
+
+
+class ModelInitializer(ModelInitializerBase):
+
+    def _load_data(self, input_file):
+        """Load sequence tuples from file."""
+        from eden.converter.fasta import fasta_to_sequence
```

Comment:
I'm not a fan of imports at a function level. There are only a few cases where this is accepted and I think this load_data function is substantial for the program. So I would move it up to the top.